packages/localize/src/LocalizeManager.js

```diff
@@ -43,16 +43,21 @@ export class LocalizeManager extends LionSingleton {
     this._setupHtmlLangAttributeObserver();
 
     if (!value.includes('-')) {
-      console.warn(`
-        Locale was set to ${value}.
-        Language only locales are deprecated, please use the full language locale e.g. 'en-GB' instead of 'en'.
-        See https://github.com/ing-bank/lion/issues/187 for more information.
-      `);
+      this.__handleLanguageOnly(value);
     }
 
     this._onLocaleChanged(value, oldLocale);
   }
 
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
Created its own method for handling this, so we can override it back to warning on our extension layer to prevent breaking change. Even so, it is private because you should not override this.